lib/tasks.ex

```diff
@@ -1,18 +1,17 @@
 defmodule Tasks do
-  @moduledoc """
-  Documentation for `Tasks`.
-  """
-
-  @doc """
-  Hello world.
-
-  ## Examples
+  def find_first(word) do
+    word = String.to_charlist(word)
```

Comment:
You could use `String.graphemes/1` here so you can return

`letter` in the helper, and not `<< helper >>`